h2o-py/h2o/plot/_plot_result.py

```diff
@@ -0,0 +1,34 @@
+# -*- encoding: utf-8 -*-
+# mutable versions of py immutable types
+class _MObject(object): pass
+class _MTuple(tuple): pass
+class _MList(list): pass
+class _MDict(dict): pass
+class _MStr(str): pass
+class Error(EnvironmentError): pass
```

Comment:
can you also use a more specific class name? It feels like you're stealing global `Error` (which doesn't exist but...).
Also `EnvironmentError` is kinda deprecated and is used for non-python errors (e.g. I/O)

I think you could use an existing H2O exception, for example the generic `H2OError`

Reply:
d5b08bb